test/end2end_test.go

```diff
@@ -39,52 +50,32 @@ func TestEndToEnd(t *testing.T) {
 		t.Fatal("Failed to create Merged_file.csv")
 	}
 
-	cmd2 := exec.Command("./tech_radar.exe", "-file", "Merged_file.csv")
-	cmd2_output, err := cmd2.Output()
+	// Generator
+	cmd2 := exec.Command("./tech_radar.exe", "generate", "Merged_file.csv")
+	_, err = cmd2.Output()
 	if err != nil {
 		t.Fatalf("%v", err)
-	} else if !strings.Contains(string(cmd2_output), "Opened csv file!") {
-		t.Errorf("Output didn't match expected. %s", string(cmd2_output))
 	}
 
-	correctHTML := `<html>
-	<head>
-		<title>Header 1</title>
-	</head>
-	<body>
-		<h1 class="pageTitle">Header 1</h1>
-		<ul>
-
-			<li>Name: TestBlip1</li>
-			<li>Quadrant: Language</li>
-			<li>Ring: Assess</li>
-			<li>Is new: true</li>
-			<li>Moved: 1</li>
-			<li>Desc: This is a description</li>
-
-			<li>Name: TestBlip2</li>
-			<li>Quadrant: Infrastructure</li>
-			<li>Ring: Adopt</li>
-			<li>Is new: false</li>
-			<li>Moved: 0</li>
-			<li>Desc: Also a description</li>
+	// Slices for assertion/checking with contains
+	correctBlipNames := []string{
+		"TestBlip1",
+		"TestBlip2",
+		"TestBlip3",
+		"TestBlip4",
+	}
```

Comment:
Assume this is temporary for tests?

Reply:
Those are names of blips from the generated test files that are used to test that merging works